internal/apps/dop/provider.go

```diff
@@ -96,10 +97,11 @@ type provider struct {
 	CICDCmsSvc            *cms.CICDCmsService                     `autowired:"erda.dop.cms.CICDCmsService"`
 	UnitTestService       *unittest.UnitTestService               `autowired:"erda.dop.qa.unittest.UnitTestService"`
 	DevFlowRule           devflowrule.Interface
-	TokenService          tokenpb.TokenServiceServer     `autowired:"erda.core.token.TokenService"`
-	ClusterSvc            clusterpb.ClusterServiceServer `autowired:"erda.core.clustermanager.cluster.ClusterService"`
-	DevFlowSvc            *flow.Service                  `autowired:"erda.apps.devflow.flow.FlowService"`
-	IssueCoreSvc          *core.IssueService             `autowired:"erda.dop.issue.core.IssueCoreService"`
+	TokenService          tokenpb.TokenServiceServer      `autowired:"erda.core.token.TokenService"`
+	ClusterSvc            clusterpb.ClusterServiceServer  `autowired:"erda.core.clustermanager.cluster.ClusterService"`
+	DevFlowSvc            *flow.Service                   `autowired:"erda.apps.devflow.flow.FlowService"`
+	IssueCoreSvc          *core.IssueService              `autowired:"erda.dop.issue.core.IssueCoreService"`
+	PublishItemSvc        *publishitem.PublishItemService `autowired:"erda.dop.publishitem.PublishItemService"`
```

Comment:
what's the difference between inject `*publishitem.PublishItemService` and `pb.PublishItemServer`?

Reply:
adjusted, remove no usage publishitemsvc